### PR TITLE
added release notes for 1.0.901-prerelease and 1.0.864.35

### PR DESCRIPTION
**Pull request by jasonstephen15**

### CODE REVIEW

File: microsoft-edge/webview2/release-notes.md [hunk 1]
```diff
@@ -19,6 +19,82 @@ The WebView2 team updates the [WebView2 SDK][NuGetGallery] on a six-week cadence
 > [!NOTE]
 > WebView2 bug fixes are either Runtime or SDK specific.  
 
+## 1.0.901-prerelease
+
+Release Date: May 31, 2021
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/release-notes.md:24]:
```suggestion
Release Date: June 1, 2021  
```

File: microsoft-edge/webview2/release-notes.md [hunk 2]
```diff
@@ -19,6 +19,82 @@ The WebView2 team updates the [WebView2 SDK][NuGetGallery] on a six-week cadence
...
+
+*   Added [IsSwipeNavigationEnabled][Webview2ReferenceWin32Icorewebview2experimentalsettings5ViewWebview210901PrereleaseGetIsswipenavigationenabled] property to enable or disable the ability of the end user to use swiping gesture on touch input enabled devices to navigate in WebView2.
+*   Added [BrowserProcessExited][Webview2ReferenceWin32Icorewebview2experimentalenvironment4ViewWebview210901PrereleaseAddBrowserprocessexited] event.
+*   Added [add_ClientCertificateRequested API][Webview2ReferenceWin32Icorewebview2experimental3ViewWebview210901PrereleaseAddClientcertificaterequested]. It allows showing a client certificate dialog prompt if desired and enables access to required metadata to replace default client certificate dialog prompt.
+*   Added iframe element support for AddHostObjectToScriptWithOrigins.
+*   Improved WebView2 startup performance and disk footprint.
+
+#### Bug fixes  
+
+*   Fix a bug where mouse left click doesn't discuss context menu. This change is Runtime-specific.
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/release-notes.md:40]: dismiss?

File: microsoft-edge/webview2/release-notes.md [hunk 3]
```diff
@@ -19,6 +19,82 @@ The WebView2 team updates the [WebView2 SDK][NuGetGallery] on a six-week cadence
...
+*   Fixed a bug where WebView2 creation fails when exe files for apps sharing the same user data folder have inconsistent version info.
+*   Fixed a bug where special browser keys like Refresh, Home, Back, etc can't be disabled by AreBrowserAcceleratorKeysEnabled. This change is Runtime-specific.
+*   Fixed a bug in WebView2 .NET controls that WebView2 windows are blank when created in the background. \([\#1077][GithubMicrosoftedgeWebviewfeedbackIssue1077]\). 
+*   Dismissing a file picker dialog by pressing 'enter' or 'esc' no longer crashes WPF applications using WebView control. \([\#1099][GithubMicrosoftedgeWebviewfeedbackIssue1099]\). 
+*   Fixed a bug that AllowSingleSignOnUsingOSPrimaryAccount (add doc link) doesn't work properly with WebView2 when a WebResourceRequested event handler is attached. This change is Runtime-specific. \([\#1183][GithubMicrosoftedgeWebviewfeedbackIssue1183]\).
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/release-notes.md:45]: `[Webview2ReferenceWin32Icorewebview2environmentoptionsGetAllowsinglesignonusingosprimaryaccount]: /microsoft-edge/webview2/reference/win32/icorewebview2environmentoptions#get_allowsinglesignonusingosprimaryaccount "get_AllowSingleSignOnUsingOSPrimaryAccount - interface ICoreWebView2EnvironmentOptions | Microsoft Docs"`

File: microsoft-edge/webview2/release-notes.md [hunk 4]
```diff
@@ -19,6 +19,82 @@ The WebView2 team updates the [WebView2 SDK][NuGetGallery] on a six-week cadence
...
+
+### General  
+
+#### Bug Fixes  
+
+*   Fixed reliability bug that could crash host app process when moving to newer Edge WebView2 Runtime version.
+*   Fixed a bug that prevented memory purge in some situations. This change is Runtime-specific.
+*   Fixed error in 818 SDK release package where project couldn't find WebView2.h file. \([\#1209][GithubMicrosoftedgeWebviewfeedbackIssue1209]\). 
+*   Fixed a bug which caused WebResourceRequested event to be dropped for some requests with binary bodies.
+*   Improve NewWIndowRequested documentation. \([\#448][GithubMicrosoftedgeWebviewfeedbackIssue448]\).
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/release-notes.md:87]:
```suggestion
*   Improve NewWindowRequested documentation. \([\#448][GithubMicrosoftedgeWebviewfeedbackIssue448]\). 
```

**Review by peiche-jessica — APPROVED** [review]

File: microsoft-edge/webview2/release-notes.md (hunk 2 above)
Comment (jasonstephen15) [reply microsoft-edge/webview2/release-notes.md:40]: yes, fixed!

File: microsoft-edge/webview2/release-notes.md (hunk 3 above)
Comment (jasonstephen15) [reply microsoft-edge/webview2/release-notes.md]: done!

**Comment by jasonstephen15** [comment]
Ready for docs team review!

**Comment by jasonstephen15** [comment]
@Reezaali this is good to go live asap, thanks!

**Review by Reezaali — APPROVED** [review]

**Pull request merged**